arviz/plots/dotplot.py

```diff
@@ -215,7 +216,11 @@ def wilkinson_algorithm(values, binwidth):
             count += 1
             if count == ndots:
                 break
-        stack_locs.append((stack_first_dot + values[count - 1]) / 2)
+        loc = float((stack_first_dot + values[count - 1]) / 2)
+        if stack_locs and (abs(loc - stack_locs[-1]) < binwidth):
+                stack_locs.append(stack_locs[-1] + binwidth)
+        else:
+            stack_locs.append(loc)
```

Comment:
Why this particular change?

Reply:
If that overlap was indented, then these are not needed